lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -83,18 +83,122 @@ describe('NxCombobox', function() {
         expect(inputElement).toHaveValue('Foo');
       });
 
-  it('sets aria-expanded on the input to true when focused',
+  it('sets aria-expanded to true on the input when initially focused and there are matches', async function() {
```

Comment:
Consider grouping all of these in a `describe('aria-expanded'` block

Reply:
> Consider grouping all of these in a `describe('aria-expanded'` block

Nested in a describe block here: [f396ba8](https://github.com/sonatype/sonatype-react-shared-components/pull/876/commits/f396ba8089204a5b15f7cdaa10c923778fa8ac6a)